src/includes/Inode.hpp

```diff
@@ -0,0 +1,66 @@
+#pragma once
+
+#include <iostream>
+#include <vector>
+#include <ctime>
+
+typedef std::string str_t;
+typedef std::time_t time_t;
+//TODO:Add a bit structure for optional bits
+
+// describes exact file or directory
+class Inode {
```

Comment:
Should we store name of file/directory in inode? I guess it can be useful for directories implementations (but I'm not sure)